Lombiq.HelpfulLibraries/Libraries/Mvc/TypedRoute.cs

```diff
@@ -114,13 +126,20 @@ private string GetRoute(string route)
                     methodParameters[index].Name,
                     ValueToString(Expression.Lambda(argument).Compile().DynamicInvoke())))
                 .Where(pair => pair.Value != null)
-                .Concat(additionalArguments);
+                .Concat(additionalArguments)
+                .ToList();
 
-            return new TypedRoute(
+            var key = string.Join(
+                separator: '|',

```

Comment:
We don't need the parameter name here - did you put it here for a specific reason?

Reply:
Interesting. What can I say? OK 😀